### PR TITLE
Create wishlist

### PR DESCRIPTION
**Pull request by studentgrad0419**
Added happy and sad path tests with "create wishlist"

Green: 100% routes.py

for next story: need to add tests for "location" response as it relies on "read wishlist"
also modified files to pass "make lint" test.

### CODE REVIEW

**Review by jay-goettelmann-nyu — APPROVED** [review]
Looks good! A couple VERY small typos and one suggestion for tear down just to truncate tables after the final test.

File: service/routes.py [hunk 1]
```diff
@@ -29,3 +29,29 @@ def index():
 ######################################################################
 
 # Place your REST API code here ...
+
+
+######################################################################
+# CREATE A NEW WISHLIST
+######################################################################
+@app.route("/wishlist", methods=["POST"])
```
[the remark this answers is not in the document]
Comment (studentgrad0419) [reply service/routes.py]: Got it, it looks like an older version of the file. It should be fixed on the newest!

File: service/routes.py [hunk 2]
```diff
@@ -29,3 +29,29 @@ def index():
...
+def create_wishlists():
+    """
+    Creates an Wishlist
+    This endpoint will create an Wishlist based the data in the body that is posted
```
[the remark this answers is not in the document]
Comment (studentgrad0419) [reply service/routes.py:41]: will be updated before merge. Thanks for the catch.

File: service/routes.py [hunk 3]
```diff
@@ -29,3 +29,29 @@ def index():
...
+    wishlist.deserialize(request.get_json())
+    wishlist.create()
+
+    # Create a message to return
+    message = wishlist.serialize()  # match test case
+
+    return make_response(
+        jsonify(message),
+        status.HTTP_201_CREATED,
+        {"Location": f"/wishlist/{wishlist.wishlist_id}"},
```
[the remark this answers is not in the document]
Comment (studentgrad0419) [reply service/routes.py]: looks like from an outdated version, it has been correctly updated.

File: tests/test_routes.py [hunk 4]
```diff
@@ -10,38 +10,46 @@
...
+from service.models import db, Wishlist, init_db
 from service.common import status  # HTTP Status Codes
-from service.routes import app
 
 DATABASE_URI = os.getenv(
     "DATABASE_URI", "postgresql://postgres:postgres@localhost:5432/postgres"
 )
 
-BASE_URL = "/accounts"
+BASE_URL = "/wishlist"
```
[the remark this answers is not in the document]
Comment (studentgrad0419) [reply tests/test_routes.py]: looks like from an outdated version, it has been correctly updated.

File: tests/test_routes.py [hunk 5]
```diff
@@ -10,38 +10,46 @@
...
     def setUpClass(cls):
         """This runs once before the entire test suite"""
+        app.config["TESTING"] = True
+        app.config["DEBUG"] = False
+        app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
+        app.logger.setLevel(logging.CRITICAL)
+        init_db(app)
 
     @classmethod
     def tearDownClass(cls):
```
[the remark this answers is not in the document]
Comment (studentgrad0419) [reply tests/test_routes.py:40]: Will add it to testing proceedure.

**Pull request merged**